schema.json

```diff
@@ -651,7 +764,49 @@
                         },
                         "engagement_social": {
                             "description": "The social engagement with the advert, presented as an array (likes/shares/comments if these features are available)",
-                            "type": "object"
+                            "type": "object",
+                            "properties": {
+                                "reactions": {
+                                    "type": "object",
+                                    "description": "This is for generic counted reactions, i.e. likes/love/haha, etc",
+                                    "patternProperties": {
+                                        "^.*$": {
+                                            "type": "number"
+                                        }
+                                    },
+                                    "additionalProperties": false
+                                },
+                                "shares": {
+                                    "type": "object",
+                                    "properties": {
+                                        "name": {
+                                            "description": "Name could be: share, retweet or post or something else platform specific",
+                                            "type": "string"
+                                        },
+                                        "value": {
+                                            "type": "number"
+                                        }
+                                    }
+                                },
+                                "comments": {
+                                    "properties": {
+                                        "name": {
+                                            "description": "Name could be: comment, quote_retweet",
+                                            "type": "string"
+                                        },
+                                        "value": {
+                                            "type": "number"
+                                        }
+                                    }
+                                },
+                                "bookmarks": {
+                                    "type": "number"
```

Comment:
we allow this to be null. that more explicitly says not supported

Reply:
In that case, we should also add the reactions from all the other platforms as null, right? I guess to keep it consistent :/

Currently, we only add `bookmarks` if it is available for the platform in question